dxr/plugins/core.py

```diff
@@ -429,6 +429,11 @@ class FileToIndex(dxr.indexers.FileToIndex):
     def __init__(self, path, contents, plugin_name, tree, vcs):
         super(FileToIndex, self).__init__(path, contents, plugin_name, tree)
         self.vcs = vcs
+        self.extension = splitext(self.path)[1].lower()
```

Comment:
Maybe we can just get away without it, as https://dxr.mozilla.org/mozilla-central/source/addon-sdk/source/test/fixtures/mofo_logo.SVG is the only uppercase one in moz-central.